scripts/capture.py

```diff
@@ -8,14 +8,29 @@
 FILE = '%H:%M:%S.png'
 
 
+def matches(frame, threshold):
```

Comment:
@Adman I believe this is not really semantically correct. A frame cannot match a threshold. I cannot think of a better name right now, but I do not really like this one.